src/agent/index.js

```diff
@@ -179,13 +174,44 @@ class Gaijin {
       .catch(err => this._showError(err, msg))
   }
   async _onReady (client) {
-    this._commandHandler = new CommandHandler(
-      this._prefix,
+    this._commandHandler = new CommandHandler({
+      prefix: this._prefix,
       client,
-      (await client.getOAuthApplication()).owner.id,
-      this._knex,
-      (await requireCommands())
-    )
+      ownerId: (await client.getOAuthApplication()).owner.id,
+      knex: this._knex,
+      replacers: new Map([
```

Comment:
Honestly, I feel these should be grouped with the commands rather than the client.

Reply:
how would they be "grouped" with commands?

Reply:
Actually, let's put them in their own folder.